src/consts/Stages.ts

```diff
@@ -225,8 +311,10 @@ export const STAGES: ReadonlyArray<STAGE> = [
         previous: "PackageApproval",
         previousEventTitle:
           "Backward Stage Change: Title V to Package Approval",
+        showStage: (request) => request?.titleV === "Yes",
```

Comment:
Comment is for the 3 lines above (311-313) -- This should now be "DraftPackageHRL" and Event Title should be updated to "to Draft Package (HRL)"